cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/common/CubeSinkConfig.java

```diff
@@ -31,9 +31,8 @@
 
   @Name(Properties.Cube.DATASET_RESOLUTIONS)
   @Description("Aggregation resolutions to be used if a new Cube dataset " +
-    "needs to be created. See Cube dataset configuration " +
-    "details for more information : http://docs.cask.co/cdap/current/en/developers-manual/building-blocks/datasets/" +
-    "cube.html#cube-configuration")
+    "needs to be created. See Cube dataset configuration details available at http://docs.cask.co for more " +
```

Comment:
if it really needs to change just for the ui, it should be done in the ui json file not in the backend